libsolidity/interface/OptimiserSettings.h

```diff
@@ -45,7 +45,7 @@ struct OptimiserSettings
 		return s;
 	}
 	/// Standard optimisations.
-	static OptimiserSettings enabled()
+	static OptimiserSettings regular()
```

Comment:
The comment calls it standard, why not go with that as a name?

Reply:
Or default?

Reply:
The problem with default is that the default is not disable the optimizer ;)
Standard sounds good, too!